src/sphinx_inline_tabs/__init__.py

```diff
@@ -13,6 +13,8 @@ def setup(app):
     # We do imports from Sphinx, after validating the Sphinx version
     from ._impl import TabContainer, TabDirective, TabHtmlTransform, TabInput, TabLabel
 
+    app.add_config_value("tabs_default_sync_behavior", "tab-title", "html", types=[str])
+    app.add_config_value("tabs_no_sync_labels", set(), "html", types=[set])
```

Comment:
IMO, this should explicitly check and present warnings when the values do not get used anywhere within the documentation set. I imagine this is tricker for `tabs_no_sync_labels`, so it's probably reasonable to skip this.

Reply:
I added better validation for `tabs_default_sync_behavior` to only allow `"tab-title"` or `"none"`. Not sure it makes sense to add any more checks for it as it's basically a global config value to specify how the extension should work. (sync enabled by default: yes or no)

As for `tabs_no_sync_labels`, I see it as something authors might want to set globally, instead of using `:no-sync:` for each tab. Besides being tricky to check if the label is used, I also wouldn't necessarily consider it an issue to add labels here in preparation for future changes.